base_app/services/yelp_service.py

```diff
@@ -2,8 +2,8 @@
 import os
 import random
 import requests
-from base_app.restaurant import Restaurant
-from base_app.restaurant_schema import RestaurantSchema
+from base_app.models.restaurant import Restaurant
+from base_app.models.restaurant_schema import RestaurantSchema
```

Comment:
Makes sense at this stage to break classes out into their respective folder types.